src/implementations/twig/components/site-header/ec-site-header.story.js

```diff
@@ -73,85 +74,155 @@ const getArgTypes = (data) => {
   if (data.login_box) {
     argTypes.show_login = {
       name: 'login',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the login box',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'true' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
     argTypes.logged = {
       name: 'logged in',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the login box for a logged in user',
       table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
         category: 'Optional',
       },
+      mapping: {
+        0: false,
+        1: true,
+      },
     };
   }
   argTypes.show_language_selector = {
     name: 'language selector',
-    type: { name: 'boolean' },
+    control: { type: 'boolean' },
     description: 'Show the language selector',
     table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: 'true' },
       category: 'Optional',
     },
+    mapping: {
+      0: false,
+      1: true,
+    },
   };
   argTypes.show_search = {
     name: 'search (Europa Search)',
-    type: { name: 'boolean' },
+    control: { type: 'boolean' },
     description: 'Show the search form',
     table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: 'true' },
       category: 'Optional',
     },
+    mapping: {
+      0: false,
+      1: true,
+    },
   };
   if (data.notification) {
     argTypes.show_notification = {
       name: 'notification',
-      type: { name: 'boolean' },
+      control: { type: 'boolean' },
       description: 'Show the notification box',
       table: {
+        type: { summary: 'boolean' },
```

Comment:
well, it seems you did this sistematically, to me it's better to distinguish between the controls that are matching a parameter and those who are only there for displaying purposes.